oidc_provider/views.py

```diff
@@ -284,18 +287,29 @@ def get(self, request, *args, **kwargs):
 
 
 class JwksView(View):
+    def _num_to_b64_string(self, value):
```

Comment:
I think pyjwt actually provides an interface for generating a JWK from a key, so I'll change that in a future PR. For now, this is just the way JWK specifies how to give your public key.